### ISSUE TITLE
Unable to optimise with Point to face loss

### ISSUE THREAD

**Issue by Sentient07** [edoRemelli/MeshSDF#10]
Hello,

Thanks for sharing this amazing work. I'm trying to minimise $\mathcal{L}_{task1}$ by defining point to face distance between reference mesh (bob) and target (spot). I have trained the network as mentioned in the readme.md. Then, I'm simply replacing the silhouette loss with [this](https://pytorch3d.readthedocs.io/en/latest/modules/loss.html#pytorch3d.loss.point_mesh_face_distance). However, i observe that mesh doesn't change and the loss is almost static (as shown below). What could be going wrong? On the other hand, I could successfully reproduce the changing topology using silhouette loss. 

Thank you in advance for your help. 
![image](https://user-images.githubusercontent.com/6195312/211006104-aa59c7f0-268a-4ca0-ae42-155afd940f88.png)

Edit: As pre-processing, I make sure all meshes have a radius of 1/1.03 similar to DeepSDF.

**Comment by bguillard** [comment]
For the record: the issue was caused by training with latent codes of size 256, which is severely over-parametrized for only 2 shapes. Switching to latent codes of dimension 2 fixed the issue.
